templates/components/telephone.build.hbs

```diff
@@ -24,5 +24,5 @@
   :placeholder="placeholder"
   tabindex="0"
 />
-<p class="text-danger" v-if="$v.value.required === false && $v.value.$dirty">Field is required.</p>
-<p class="text-danger" v-if="$v.value.phone === false && $v.value.$dirty">Phone could contain <b>; , . ( ) - + SPACE * #</b> and numbers.</p>
+<p class="text-danger" v-if="$v.value.required === false && $v.value.$dirty">{{T "widgets.formBuilder.dataSource.errors.required"}}</p>
+<p class="text-danger" v-if="$v.value.phone === false && $v.value.$dirty">{{T "widgets.formBuilder.dataSource.telephone.instruction"}}</p>
```

Comment:
Please share a screenshot confirming that `{{T "widgets.formBuilder.dataSource.telephone.instruction"}}` works.

Reply:
> Пожалуйста, поделитесь снимком экрана, подтверждающим, что это `{{T "widgets.formBuilder.dataSource.telephone.instruction"}}`работает.

```suggestion
<p class="text-danger" v-if="$v.value.phone === false && $v.value.$dirty">{{T "widgets.formBuilder.dataSource.telephone.instruction"}}</p>
![tel-error](https://user-images.githubusercontent.com/86256215/134631899-10431e73-e50b-460e-b8ee-afccda7b588a.jpg)

```

Reply:
> Пожалуйста, поделитесь снимком экрана, подтверждающим, что это `{{T "widgets.formBuilder.dataSource.telephone.instruction"}}`работает.
![tel-error](https://user-images.githubusercontent.com/86256215/134632077-22a57abf-b20e-46f5-8632-df2c3f0b8436.jpg)

Reply:
👍🏻